src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -3746,7 +3743,12 @@ class SymbolTableVisitor : public CommonVisitor<SymbolTableVisitor> {
     void visit_Import(const AST::Import_t &x) {
         ASR::symbol_t *t = nullptr;
         std::string rl_path = get_runtime_library_dir();
-        std::vector<std::string> paths = {rl_path, parent_dir};
+        std::vector<std::string> paths;
+        for (auto &path:import_paths) {
+            paths.push_back(path);
+        }
+        paths.push_back(rl_path);
+        paths.push_back(parent_dir);
```

Comment:
This logic seems to repeat at two places, so we could probably wrap it into a function which returns a vector of strings. I would like to know if the current approach is fine or if we should wrap this code into a function.